docs/tasks/access-application-cluster/create-external-load-balancer.md

```diff
@@ -97,20 +99,34 @@ which should produce output like this:
 
 The IP address is listed next to `LoadBalancer Ingress`.
 
-## Annotation to modify the LoadBalancer behavior for preservation of Source IP
-In 1.5, a Beta feature has been added that changes the behavior of the external LoadBalancer feature.
+## Modify the LoadBalancer behavior for preservation of Source IP
+
+Due to the implementation of this feature, the source IP for sessions as seen in the target
+container will *not be the original source IP* of the client. This is the default behavior as
+of Kubernetes v1.7. However, starting in v1.5, an optional beta feature has been added that
```

Comment:
Replace starting with "This is the default..."

```
To enable preservation of the client IP, the following field can be configured in the service spec:
```

you can delete the version stuff as you already cover it below in the beta etc discussion.